lib/improver/tests/nbhood/circular_kernel/test_CircularProbabilities.py

```diff
@@ -49,24 +49,184 @@ class Test__repr__(IrisTest):
 
     def test_basic(self):
         """Test that the __repr__ returns the expected string."""
-        result = str(CircularNeighbourhood())
-        msg = '<CircularNeighbourhood: unweighted_mode: False>'
+        result = str(CircularProbabilities())
+        msg = '<CircularProbabilities: weighted_mode: True>'
         self.assertEqual(str(result), msg)
 
 
-class Test_apply_circular_kernel(IrisTest):
+class Test_circular_kernel(IrisTest):
```

Comment:
Happy these tests have been copied from pr #213 and I've already reviewed these.